test/parallel_api/numeric/numeric.ops/reduce_by_segment.pass.cpp

```diff
@@ -346,13 +346,22 @@ run_test()
 int
 main()
 {
+    // On Windows, we observe incorrect results with this test with a specific compilation order of the
```

Comment:
I wonder whether we even need this logic. Was there a specific reason for ordering these tests in the current way? If not, couldn't we simply move the `MatrixPoint` test in general?

Reply:
Sure, we could do this, but it is not very satisfying to move tests merely because it suppresses an error, it feels like sweeping dirt under the rug.  The advantage of this is that it records that there is an error, and the situation where it exists.  Obviously, we are also planning to pursue it with the compiler team as well.

I don't think there is anything stopping us from just changing the order of the tests across the board, but that would lose the information and ability to reproduce the issue via these tests in the future.

Reply:
That makese sense, thanks.